core/unix/src/TUnixSystem.cxx

```diff
@@ -40,10 +40,15 @@
 #include <algorithm>
 #include <atomic>
 
+#ifdef __linux__
+#include <syscall.h>
+#endif
+
 //#define G__OLDEXPAND
 
 #include <unistd.h>
 #include <stdlib.h>
+#include <poll.h>
```

Comment:
I think this is not available on windows. Would using select be an option?  Otherwise we need to hide this (and code using it) behind 
```ifndef R__WIN32```